crowdin.yml

```diff
@@ -0,0 +1,127 @@
+#
+# Your Crowdin credentials
+#
+"project_id_env": CROWDIN_PROJECT_ID
+"api_token_env": CROWDIN_PERSONAL_TOKEN
+"base_path" : "."
+"base_url" : "https://metamask.crowdin.com"
+
+#
+# Choose file structure in Crowdin
+# e.g. true or false
+#
+"preserve_hierarchy": true
+
+#
+# Files configuration
+#
+files: [
+ {
+  #
+  # Source files filter
+  # e.g. "/resources/en/*.json"
+  #
+  "source" : "app/_locales/en/messages.json",
+
+  #
+  # Where translations will be placed
+  # e.g. "/resources/%two_letters_code%/%original_file_name%"
+  #
+  "translation" : "/app/_locales/%two_letters_code%/%original_file_name%",
+
+  #
```

Comment:
we can probably delete all the commented out configuration for now?

Reply:
I guess this was just how it was initialised? I thought maybe it could be useful for reference, but I could absolutely delete the comments and then we could just reference back to this (or their docs) if we ever need to adjust.